process/process.lua

```diff
@@ -367,11 +370,20 @@ function process.handle(msg, _)
       function (msg)
         return msg.Tags.Type == "Process" and Owner == msg.From 
       end,
-      require('.boot')(ao)
+      function ()
+        require('.boot')(ao)    
+        -- AR.IO PROCESS CODE
+        require(".src.init").init()
+        -- load all the initial state once
+        require(".state.init").init()
+      end
     )
   end
 
-  Handlers.append("_default", function () return true end, require('.default')(insertInbox))
+  Handlers.append("_default", function () return true end, function (msg)
+    print('{ "_e": 1, "Message-Id": "' .. msg.Id .. '", "From": "' .. msg.From .. '", "Action": "' .. msg.Action .. '", "Timestamp": ' .. tonumber(msg.Timestamp) .. ', "Default-Handler": true}')
```

Comment:
We should guard against nil Actions. I sent a test message with no Action to 4U4... and got this log in the CU (and no event):
```
2025-02-19T23:00:58.857Z ao-cu:Ariels-Mac-mini.local:readState:evaluate [info]: Error occurred when applying message "Scheduled Message aALxjs6dcHhtVBE7QVB7DvNBj55s0RF9SZ2dLSh76cA 1740006047330:53" to process "4U4klqiTzMSnBy6SUpYScfO8kp9i5QgZ5WeYQaCm978": "Pruning state at timestamp: 1740006047330
Error handling message
[string ".process"]:767: attempt to concatenate a nil value (field 'Action')

stack traceback:
	[string ".process"]:901: in function '.process.handle'

error:
```

Reply:
a3f3523